pkg/process/checks/process.go

```diff
@@ -592,6 +602,9 @@ func skipProcess(
 		// This means short-lived processes (<2s) will never be captured.
 		return true
 	}
+	if zombiesIgnored && fp.Stats != nil && fp.Stats.Status == "Z" {
```

Comment:
Nit: pretty obvious by reading the code, but may be helpful to add a small comment and reference procfs doc here.